src/server/api/campaign.js

```diff
@@ -149,6 +149,26 @@ export const resolvers = {
         .limit(1)
       return contacts.length > 0
     },
+    hasUnsentInitialMessages: async (campaign) => {
+      const contacts = await r.knex('campaign_contact')
+        .select('id')
+        .where({
+          campaign_id: campaign.id,
+          message_status: 'needsMessage',
+          is_opted_out: false
+         })
+        .limit(1)
+      return contacts.length > 0
+    },
+    customFields: async (campaign) => {
```

Comment:
This is a duplicate key of the function below, which should be preferred since it uses cache when available.  Is it broken?  Then we should fix in dbCustomFields.

Reply:
good catch - merge conflict issue